DateMeter.c

```diff
@@ -0,0 +1,47 @@
+/*
+htop - DateMeter.c
+(C) 2004-2011 Hisham H. Muhammad
```

Comment:
I think you should put your name here (or something like "The htop contributors")

Reply:
Added myself and updated the years. I let Hisham remain in the files, since the files contain a significant amount of his code, copied from ClockMeter and UptimeMeter.